src/Akka.Cluster.Hosting/ClusterClientOptions.cs

```diff
@@ -0,0 +1,80 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using Akka.Actor;
+using Akka.Cluster.Tools.Client;
+using Akka.Remote;
+
+namespace Akka.Cluster.Hosting
+{
+    public class ClusterClientOptions
+    {
+        /// <summary>
+        /// Actor paths of the <see cref="ClusterReceptionist"/> actors on the servers (cluster nodes) that the client will try to contact initially.
+        /// </summary>
+        public HashSet<ActorPath> InitialContacts { get; set; }
```

Comment:
Have this default to `/system/receptionist` - eliminates one big papercut that way.

Reply:
Ah, I see - these have to be remote `ActorPath`s.... Any ideas how we can make that easier?

Reply:
Not that I can see, the `ClusterClient` would need to know where the receptionist is to gather more contact informations.